gctscript/wrappers/validator/validator.go

```diff
@@ -0,0 +1,199 @@
+package validator
+
+import (
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/orderbook"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/withdraw"
+	"github.com/thrasher-corp/gocryptotrader/gctscript/modules"
+)
+
+func (w Wrapper) Exchanges(enabledOnly bool) []string {
+	if enabledOnly {
+		return []string{
+			"hello world",
+		}
+	}
+	return []string{
+		"nope",
+	}
+}
+
+func (w Wrapper) IsEnabled(exch string) (v bool) {
+	if exch == exchError.String() {
+		return
+	}
+	return true
+}
+
+func (w Wrapper) Orderbook(exch string, pair currency.Pair, item asset.Item) (*orderbook.Base, error) {
```

Comment:
Please add comments for this functions

Reply:
aww man I tried so hard to get all exported comments on this PR and still missed heaps